graviti/dataframe/column/series.py

```diff
@@ -115,7 +115,12 @@ def __getitem__(self, key: int) -> Any:
         return self._data[key]
 
     def __delitem__(self, key: Union[int, slice]) -> None:
-        self._data.__delitem__(key)
+        if self._root is not None:
```

Comment:
ref commit message:
`feat(dataframe): forbid deleting items by loc for non-root Containers`